### ISSUE TITLE
tricks to reach 0.909?

### ISSUE THREAD

**Issue by fanskyer** [NVIDIA/DeepRecommender#13]
i tried to follow the example on full dataset and open the dense refeeding trick every iteration. make the training 36 iterations [seems do not change much after that?], but my best result on validation set is 0.914, some gap to the reported 0.91. is there other tricks to duplicate the result? thanks!

**Comment by okuchaiev** [comment]
We did train for longer than 36 epochs. But we monitored *validation* error and then picked a checkpoint (was around step 40 I think) with the best validation error.
